src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -3376,22 +3377,22 @@ public override void PostVisit(DottedNameNode node)
                     }
                     else
                     {
-                        SetDottedNameError(node, TexlStrings.ErrInvalidName);
+                        SetDottedNameError(node, TexlStrings.ErrInvalidName, nodeNameStr);
                         return;
                     }
                 }
                 else if (leftType.IsOptionSet || leftType.IsView)
                 {
                     if (!leftType.TryGetType(nameRhs, out typeRhs))
                     {
-                        SetDottedNameError(node, TexlStrings.ErrInvalidName);
+                        SetDottedNameError(node, TexlStrings.ErrInvalidName, nodeNameStr);
                         return;
                     }
                 }
                 else if (leftType.IsAttachment)
                 {
                     // Error: Attachment Type should never be the left hand side of dotted name node
-                    SetDottedNameError(node, TexlStrings.ErrInvalidName);
+                    SetDottedNameError(node, TexlStrings.ErrInvalidName, nodeNameStr);
```

Comment:
>nodeNameStr [](http://example.com/codeflow?start=73&length=11)

Not all of these errors are for looking up the RHS. 

The basic rule is that if we lookup string x and can't find it, then issue an error for string x.  Line 3380 is a good case of that. 

In the other cases (like this one) - we should probably leave unchanged.